client/src/pages/ProfileDetails.js

```diff
@@ -94,8 +96,23 @@ const ProfileDetails = function () {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    console.log(selectDropIn, selectDropOff);
-    //api call
+    fetch("/request/create", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        sitter_id: params.id,
+        start: selectDropIn,
+        end: selectDropOff,
+      }),
+    }).then(() => {
+      fetch("/profile/me").then((profile) => {
```

Comment:
I'm wondering why we have to update the context after sending a request?

Reply:
Yeah, that's the purpose of updating the context, because if we don't then when the user goes to My Sitters the request doesn't appear

Reply:
Replaced with the pullProfile function added in another pull request